ethportal-peertest/src/scenarios/gossip.rs

```diff
@@ -202,18 +320,42 @@ pub async fn test_gossip_dropped_with_find_content(peertest: &Peertest, target:
         .await
         .unwrap();
 
-    // Store accumulator_1 locally in client that is not connected to the network
-    let (acc_key_1, acc_value_1) = fixture_epoch_acc_1();
+    // Store block_1 locally in client that is not connected to the network
+    let (header_key_1, header_value_1) = fixture_header_by_hash_with_proof_15040708();
+    let (body_key_1, body_value_1) = fixture_block_body_15040708();
+    let (receipts_key_1, receipts_value_1) = fixture_receipts_15040708();
+    let store_result = HistoryNetworkApiClient::store(
+        &fresh_target,
+        header_key_1.clone(),
+        header_value_1.encode(),
+    )
+    .await
+    .unwrap();
+    assert!(store_result);
     let store_result =
-        HistoryNetworkApiClient::store(&fresh_target, acc_key_1.clone(), acc_value_1.encode())
+        HistoryNetworkApiClient::store(&fresh_target, body_key_1.clone(), body_value_1.encode())
             .await
             .unwrap();
     assert!(store_result);
+    let store_result = HistoryNetworkApiClient::store(
+        &fresh_target,
+        receipts_key_1.clone(),
+        receipts_value_1.encode(),
+    )
+    .await
+    .unwrap();
+    assert!(store_result);
 
-    // Store accumulator_2 locally in target
-    let (acc_key_2, acc_value_2) = fixture_epoch_acc_2();
+    // Store block_2 minus receipts locally in target
+    let (header_key_2, header_value_2) = fixture_header_by_hash_with_proof_15040709();
```

Comment:
I don't think you actually need to store the header locally in every target. As long as it's available in any single node, it should be available to the others for validation.

Reply:
I am confused, am I not just storing the header in target?